<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblLogo_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-10</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Inserir_um_título-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtcpfconsulta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblenderecoconsulta">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Endereço:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNomeconsulta">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome Completo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtenderecoconsulta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblcpfconsulta">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CPF:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultarcliente">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 169, 39);</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNomeconsulta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnlimpar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFundo_2">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>-20</y>
      <width>781</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblnomeanimal2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome Animal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtnomeanimal2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblidanimal2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtidanimal2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>560</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btndeletarcliente">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 0, 4);
background-color: rgb(241, 105, 105);</string>
    </property>
    <property name="text">
     <string>Deletar Cadastro</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblemail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtemail">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txttelefone">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbltelefone">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telefone:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnvoltarmenucliente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 165, 165);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>botao-voltar.png</normaloff>botao-voltar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <zorder>lblFundo_2</zorder>
   <zorder>lblLogo_2</zorder>
   <zorder>txtcpfconsulta</zorder>
   <zorder>lblenderecoconsulta</zorder>
   <zorder>lblNomeconsulta</zorder>
   <zorder>txtenderecoconsulta</zorder>
   <zorder>lblcpfconsulta</zorder>
   <zorder>btnConsultarcliente</zorder>
   <zorder>txtNomeconsulta</zorder>
   <zorder>btnlimpar</zorder>
   <zorder>lblnomeanimal2</zorder>
   <zorder>txtnomeanimal2</zorder>
   <zorder>lblidanimal2</zorder>
   <zorder>txtidanimal2</zorder>
   <zorder>btndeletarcliente</zorder>
   <zorder>lblemail</zorder>
   <zorder>txtemail</zorder>
   <zorder>txttelefone</zorder>
   <zorder>lbltelefone</zorder>
   <zorder>btnvoltarmenucliente</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
